docs/IE_DG/Extensibility_DG/Custom_ONNX_Ops.md

```diff
@@ -0,0 +1,59 @@
+# Custom ONNX operators {#openvino_docs_IE_DG_Extensibility_DG_Custom_ONNX_Ops}
+
+ONNX importer provides mechanism to register custom ONNX operators based on predefined or user defined nGraph operations.
+The function responsible for registering a new operator is called `ngraph::onnx_import::register_operator` and is defined in `onnx_import/onnx_utils.hpp`.
+
+## Registering custom ONNX operator based on predefined nGraph operations
+
+As an example, let's register 'CustomRelu' ONNX operator in a domain called 'com.example'.
```

Comment:
I'd rewrite this section to something more structured and easy to navigate through:

The steps below explain how to register a custom ONNX operator, for example, `CustomRelu`, in a domain called `com.example`.